modules/src/index.js

```diff
@@ -0,0 +1,54 @@
+import * as task from './status.js';
+import './style.css';
+
+let list = [];
+const listEl = document.querySelector('ul');
+
+function todoList() {
+  if (window.localStorage.getItem('localTasks')) {
+    const localTasks = window.localStorage.getItem('localTasks');
+    list = JSON.parse(localTasks);
+  }
+  document.querySelector('.list-1').innerHTML = '';
+  list.forEach((item) => {
+    const taskElement = document.createElement('li');
+    taskElement.classList.add('task');
+    if (item.isCompleted) {
+      taskElement.classList.add('completed');
+    }
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.classList.add('task-check');
+    checkbox.addEventListener('click', () => {
+      task.status(item, list);
+      todoList();
+    });
```

Comment:
- [ ] (Required) When I change the `Completed` status of a task and I reload the page, the status update is lost. Although the data is stored in the `localStorage,` there is no function or line of code to fetch data from the `localStorage`. I would like to advise you to always check if there is data in the localStorage upon reload of the webpage. If there is data stored in there, you can retrieve it and assign the `list` array to the data retrieved.  If there is none stored, you can set the `list` array to an empty array.